src/content/roadmap/single-slot-finality/index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Single slot finality
+description: Explanation of single slot finality
+lang: en
+summaryPoints:
+  - Single slot finality means that transactions become irreversible almost instantly
+  - Near-instant finality means users and apps can be immediately sure of the permanence of their transactions.
+  - Several types of attack rely upon the ability to reorganize blocks - single slot finality eliminates them.
+---
+
+# Single slot finality {#single-slot-finality}
+
+Today it takes about 15 minutes for an Ethereum block to finalize. However, there are several improvements that can be made to the consensus mechanism that will make validating blocks much more efficient and allow the time-to-finality to decrease dramatically. Instead of waiting for fifteen minutes, blocks could be finalized in the same slot in which it is proposed. This is known as **single slot finality (SSF)**.
```

Comment:
```suggestion
It takes about 15 minutes for an Ethereum block to finalize. However, we can make Ethereum's consensus mechanism validate blocks more efficiently and decrease time-to-finality dramatically. Instead of waiting for fifteen minutes, blocks could get proposed and finalized in the same slot. This concept is known as **single slot finality (SSF)**.
```